### PR TITLE
Fix api/ and sdk/ package tests

### PR DESCRIPTION
**Pull request by miagilepner**

### CODE REVIEW

File: sdk/helper/pluginutil/run_config_test.go [hunk 1]
```diff
@@ -322,7 +322,7 @@ func TestMakeConfig(t *testing.T) {
 			responseWrapInfoTimes: 0,
 
 			mlockEnabled:      false,
-			mlockEnabledTimes: 1,
```
Comment (miagilepner) [inline sdk/helper/pluginutil/run_config_test.go:325]: @tomhjp I'm not confident about the changes in this file, would appreciate if you could take a look to make sure that this is the intended behavior

**Review by tomhjp — APPROVED** [review]
LGTM, thanks!

File: sdk/helper/pluginutil/run_config_test.go (hunk 1 above)
Comment (tomhjp) [reply sdk/helper/pluginutil/run_config_test.go:325]: Yeah this is correct - it checks once for the env var, and once for setting the IPC_LOCK config option in `plugincontainer`.

Comment (tomhjp) [reply sdk/helper/pluginutil/run_config_test.go:325]: And `GRPCBrokerMultiplex` recently became the default when using a container image.

**Pull request merged**